pilot/pkg/model/telemetry_test.go

```diff
@@ -315,6 +315,19 @@ func TestAccessLoggingWithFilter(t *testing.T) {
 	}
 }
 
+func newTracingConfigWithRequestIDForTraceSampling(providerName string, disabled bool) *TracingConfig {
+	return &TracingConfig{
+		Provider:                     &meshconfig.MeshConfig_ExtensionProvider{Name: providerName},
+		Disabled:                     disabled,
+		UseRequestIDForTraceSampling: true,
+	}
+}
+
+const (
+	reportingEnabled  = true
+	reportingDisabled = !reportingEnabled
```

Comment:
i think i've confused myself.  should these values be reversed?  `reportingEnabled = false`, so that we get something like:

```
newTracingConfig("foo", reportingEnabled) => &TracingConfig{Provider: "foo", Disabled: false...}
```

Reply:
fixed